scripts/keys.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What is this for? remove if unneeded, and add to `.gitignore` if it is different for everyone.

Reply:
used to load private/public key value pairs